src/components/checkbox/__tests__/index.spec.js

```diff
@@ -0,0 +1,94 @@
+import React, {useState, useCallback} from 'react';
+import {render, fireEvent} from '@testing-library/react-native';
+import Checkbox from '../index';
+
+export const testID = 'checkbox';
```

Comment:
Why are you exporting the testID?

Reply:
I think I've put the `TestCase` in a different file and it was copy-pasted, 10x